src/familybot/lib/database.py

```
@@ -366,4 +366,4 @@ def _mark_migration_run(migration_name: str) -> None:
             )
             conn.commit()
     except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)